course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -215,7 +215,7 @@
                 }
             });
 
-            modalInstance.result.then(function modalConfirmed() {
+            $scope.modalInstance.result.then(function modalConfirmed() {
```

Comment:
Do we need a `finally()` here that [removes this from the controller scope](https://github.com/Harvard-University-iCommons/canvas_account_admin_tools/blob/task/elliottyates/tlt-2119/course_info_add_multiple_people_jasmine_tests/course_info/static/course_info/js/controllers/PeopleController.js#L252), then, like we do for `$scope.confirmRemoveModalInstance`?